controller/src/main/java/org/jboss/as/controller/ControlledProcessState.java

```diff
@@ -38,10 +38,18 @@
          */
         STARTING("starting"),
         /**
+         * @deprecated see {@link #OK}
+         * @see {@link org.jboss.as.controller.operations.common.ProcessStateAttributeHandler}
+         * for how the new values are mapped to the old attributes for compatibility.
          * The process is started, is running normally and has a runtime state consistent with its persistent configuration.
          */
         RUNNING("running"),
         /**
+         * replaces #RUNNING
+         * The process is started, is operating normally and has a runtime state consistent with its persistent configuration.
+         */
+        OK("ok"),
```

Comment:
This is a problem. ControlledProcessStateService exposes this enum, and it may be used by subsystem to check for state transitions. ControlledProcessStateService allows for registration of property change listeners for that purpose. That stuff is marked deprecated, but I think we can only break it in an EAP major.

I know I've pointed out that API a few times to folks who are looking for some way to track the server lifecycle. They may break if a new OK value is introduced.

Sorry; I should have mentioned this before. :( I was thinking in terms of the external API, not the internals.